vitrivr-engine-query/src/main/kotlin/org/vitrivr/engine/query/parsing/QueryParser.kt

```diff
@@ -66,7 +67,14 @@ class QueryParser(val schema: Schema) {
         /* Extract necessary information. */
         val operation = description.operations[operatorName] as? RetrieverDescription ?: throw IllegalArgumentException("Operation '$operatorName' not found in information need description.")
         val input = description.inputs[operation.input] ?: throw IllegalArgumentException("Input '${operation.input}' for operation '$operatorName' not found")
-        val field = operation.field?.let { this.schema[it] }
+        val fieldAndAttributeName: Pair<String,String?> = if(operation.field.contains(".")){
+            val f = operation.field.split(".").firstOrNull() ?: throw IllegalArgumentException("Field name in dot notation FIELD.ATTRIBUTE requires both, field and attribute")
+            val a = operation.field.split(".").lastOrNull() ?: throw IllegalArgumentException("Field name in dot notation FIELD.ATTRIBUTE requires both, field and attribute")
```

Comment:
This will not behave in the way you want if there is more than one `.` in the name.

Reply:
Fair enough, it should be addressed directly as the first and second elements after the split. Other notations are (currently) not supported